src/D06/Group.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+
+namespace aoc.D06
+{
+	public class Group
+	{
+		List<HashSet<char>> _responses = new List<HashSet<char>>();
+
+		public Group(string[] lines)
+		{
+			foreach (var line in lines)
+				_responses.Add(new HashSet<char>(line.ToCharArray()));
+		}
+
+		public static int GetAnyoneAnsweredYesCount(List<Group> groups)
```

Comment:
I would use IEnumerable instead of List - to avoid mutation

Reply:
@Jaxwood I changed the signature of both methods to `IEnumerable` but how doe sit improve anything? please elaborate how it impacts immutability